variants/vmware-k8s-1.19/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+# This is the vmware-k8s-1.19 variant.  "." is not allowed in crate names, but
+# we don't use this crate name anywhere.
+name = "vmware-k8s-1_19"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant]
+image-format = "vmdk"
+supported-arches = ["x86_64"]
+kernel-parameters = [
+    "console=ttyS0",
+    "console=tty1",
```

Comment:
Should probably add `systemd.unified_cgroup_hierarchy=1` like 1.20.  (I'm also assuming we'll update this variant to 1.20.)

Reply:
Or not!  https://github.com/bottlerocket-os/bottlerocket/pull/1533  :D